DeploymentCloud/Deployment.Common/CosmosDB/commons.json

```diff
@@ -701,5 +717,171 @@
+						"type" : "SimpleBox",
+						"position" : "FirstRow"
+					}, 
+					{
+						"name" : "averageEvents",
+						"displayName" : "Avg. Events/Minute",
+						"data" : "events_average",
+						"formatter" : "longint",
+						"type" : "SimpleBox",
+						"position" : "FirstRow"
+					}
+				],
+				"initParameters" : {
+					"widgetSets" : [ 
+						"direct"
+					],
+					"jobNames" : {
+						"type" : "getCPSparkJobNames"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name" : "DataXBatch",
+		"content" : {
+			"name" : "${sparkJobName}",
+			  "cluster": "$sparkName",
+			"options" : {
+				"name" : "${sparkJobName}",
+				"file" : "wasbs:///datax/bin/datax-host_2.3_2.11-1.1.0-SNAPSHOT.jar",
+				"className" : "datax.app.BatchApp",
+				"args" : [ 
+					"conf=${sparkJobConfigFilePath}", 
+					"driverLogLevel=${sparkJobDriverLogLevel}"
+				],
+				"jars" : [ 
+					"wasbs:///datax/bin/datax-core_2.3_2.11-1.1.0-SNAPSHOT.jar", 
+					"wasbs:///datax/bin/datax-utility_2.3_2.11-1.1.0-SNAPSHOT.jar", 
+					"wasbs:///datax/bin/applicationinsights-core-2.2.1.jar", 
+					"wasbs:///datax/bin/azure-documentdb-1.16.1.jar", 
+					"wasbs:///datax/bin/azure-eventhubs-1.2.1.jar", 
+					"wasbs:///datax/bin/azure-eventhubs-spark_2.11-2.3.6.jar", 
+					"wasbs:///datax/bin/azure-keyvault-webkey-1.1.jar", 
+					"wasbs:///datax/bin/datax-keyvault_2.3_2.11-1.1.0-SNAPSHOT-with-dependencies.jar", 
+					"wasbs:///datax/bin/java-uuid-generator-3.1.5.jar", 
+					"wasbs:///datax/bin/proton-j-0.31.0.jar",
+					"wasbs:///datax/bin/scala-java8-compat_2.11-0.9.0.jar"
+				],
+				"driverMemory" : "1024m",
+				"executorCores" : 2,
+				"numExecutors" : "${sparkJobNumExecutors}",
+				"driverCores" : 1,
+				"executorMemory" : "${sparkJobExecutorMemory}",
+				"conf" : {
+				  "spark\uff0eexecutor\uff0eextraJavaOptions": "-verbose:gc -XX:+UseG1GC -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintFlagsFinal -XX:+PrintReferenceGC -XX:+PrintAdaptiveSizePolicy -XX:+UnlockDiagnosticVMOptions -XX:+G1SummarizeConcMark -XX:+AlwaysPreTouch -XX:+PrintCodeCache -XX:ReservedCodeCacheSize=240m -XX:+UseCodeCacheFlushing",
+				  "spark\uff0eyarn\uff0eam\uff0eattemptFailuresValidityInterval": "1h",
+				  "spark\uff0eyarn\uff0eexecutor\uff0efailuresValidityInterval": "1h",
+				  "spark\uff0eblacklist\uff0eenabled": "true",
+				  "spark\uff0edriver\uff0euserClassPathFirst": true,
+				  "spark\uff0eexecutor\uff0euserClassPathFirst": true,
+				  "spark\uff0especulation": true,
+				  "spark\uff0especulation\uff0emultiplier": 3,
+				  "spark\uff0especulation\uff0equantile": 0.9,
+				  "spark\uff0eexecutorEnv\uff0eDATAX_DEFAULTVAULTNAME" : "$sparkKVName",
+				  "spark\uff0eexecutorEnv\uff0eDATAX_APPINSIGHT_KEY": "$appinsightkey",
+				  "spark\uff0eexecutorEnv\uff0eDATAX_APPNAME": "${sparkJobName}",
+				  "spark\uff0eexecutorEnv\uff0eDATAX_LOGLEVEL": "${sparkJobExecutorLogLevel}",
+				  "spark\uff0eexecutorEnv\uff0eDATAX_CHECKPOINTENABLED": "true",
```

Comment:
\ff0e characters seem weird here.  Should this be a \?

Reply:
It should be encoded this way to convert a bson document for MongoDB,